### ISSUE TITLE
[Devops] Provide docker compose files with all needed deps

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot#269]
- run db server
- run backend app

Use the setup for expert consultation project as an example for desired setup: https://github.com/code4romania/expert-consultation-api/blob/develop/etc/docker/README.md

**Comment by mirceachira** [comment]
I'd like to work on this if possible

**Comment by aniri** [comment]
> I'd like to work on this if possible
> 
> Could you provide a bit more context on the requirements?
> Do you want a single docker-compose for development for now or a setup similar to expert consulting where you have one for development and one for production?

@mirceachira sorry, I somehow missed your question :/ Yes, for now we only need a docker-compose for development. 

It should start a mssql server instance, import db test data and start the backend app from the image on docker-hub (implemented here https://github.com/code4romania/monitorizare-vot/issues/270). I have also added some test db data here: https://github.com/code4romania/monitorizare-vot/tree/develop/docs/dummy-db-data

**Comment by GabrielMajeri** [comment]
@mirceachira Are you still working on this issue, or is it available?

**Comment by mirceachira** [comment]
@GabrielMajeri if you can pick it up and it's ok with @aniri then please go-ahead
If not, I can continue on it asap. :)

**Comment by aniri** [comment]
Hey @GabrielMajeri & @mirceachira thanks for your interest in this issue :smiley: It is not urgent, but a nice to have. So if any of you can handle it, it would be great. You can decide which one picks it up :)

**Comment by GabrielMajeri** [comment]
I'm first trying to clarify the details of this issue, to make sure I understand what we're implementing :)

- The DB server can use Microsoft's official [mssql container image](https://hub.docker.com/_/microsoft-mssql-server)

- The backend app will use the Docker image from #270, so this issue is currently blocked on that

- How are we going to load the seed data into the database? Do we already have functions in the code for importing the `.csv` files, or will this have to be implemented?

**Comment by aniri** [comment]
hey @GabrielMajeri 

- #270 has PR open, so it's pretty much ready :)
- there was something in place for adding some seed data in the db, but I'm not sure if it's still available or something new should be implemented :(
